tools/azure-sdk-tools/devtools_testutils/resource_testcase.py

```diff
@@ -71,10 +71,11 @@ def create_resource(self, name, **kwargs):
             parameters = {"location": self.location}
             expiry = datetime.datetime.utcnow() + self.delete_after_tag_timedelta
             parameters["tags"] = {"DeleteAfter": expiry.replace(microsecond=0).isoformat()}
-            parameters["tags"]["BuildId"] = os.environ.get("BUILD_BUILDID", None)
-            parameters["tags"]["BuildJob"] = os.environ.get("AGENT_JOBNAME", None)
-            parameters["tags"]["BuildNumber"] = os.environ.get("BUILD_BUILDNUMBER", None)
-            parameters["tags"]["BuildReason"] = os.environ.get("BUILD_REASON", None)
+
+            parameters["tags"]["BuildId"] = os.environ.get("BUILD_BUILDID", "local")
```

Comment:
I honestly don't mind, but I feel like `local` setting will be easily recognizable as well.

Reply:
LGTM